File: prime.py
def isPrime(x):
    
    if x < 2:
        return 0
    
    for i in range(2, x):
        if (x % i) == 0:
            return 0
    
    return 1;




counter = 0

for i in range(1, 20000):
    if isPrime(i) == 1:
        counter += 1


print("Total: ", counter)

File: prime.go
package main

import (
  "fmt"
)

func isPrime(x int) int {
  if x < 2 {
    return 0
  }

  for i := 2; i < x; i++ {
    if x%i == 0 {
      return 0
    }
  }

  return 1

}

func main() {
  counter := 0

  for i := 1; i < 20000; i++ {
    if isPrime(i) == 1 {
      counter++
    }
  }

  fmt.Printf("Total: %d\n", counter)
}
